ord_schema/updates_test.py

```diff
@@ -23,16 +23,20 @@
 
 class UpdatesTest(absltest.TestCase):
     def test_resolve_names(self):
+        roundtrip_smi = lambda smi: Chem.MolToSmiles(Chem.MolFromSmiles(smi))
         message = reaction_pb2.Reaction()
         message.inputs['test'].components.add().identifiers.add(
             type='NAME', value='aspirin')
         self.assertTrue(updates.resolve_names(message))
+        resolved_smi = roundtrip_smi(
+            message.inputs['test'].components[0].identifiers[1].value)
+        self.assertEqual(resolved_smi, roundtrip_smi('CC(=O)Oc1ccccc1C(O)=O'))
         self.assertEqual(
-            message.inputs['test'].components[0].identifiers[1],
-            reaction_pb2.CompoundIdentifier(
-                type='SMILES',
-                value='CC(=O)OC1=CC=CC=C1C(=O)O',
```

Comment:
Since these can differ, is it important to have the details specify which service was used?

Reply:
I was wondering if you thought that would be worthwhile; I think it's less important to note _how_ it was automatically resolved (assuming it was a somewhat reputable service). The intent is for the user to manually check whether the structure makes sense when using the webform